ws_fnc/AI/fn_createGarrison.sqf

```diff
@@ -204,23 +217,36 @@ for "_x" from 1 to _int do {
 	_u setPosATL _bp;
 	[_u] joinsilent _grp; //otherwise the side might be wrong
 	dostop _u;
-	//_u disableAI "PATH";
+	// _u disableAI "PATH";
 
 	_u spawn ws_fnc_setInsidePos; // SetInsidePos is fairly expensive, thus spawned
 
 	// Set new variables
 	_u setVariable ["ws_bpos",_bp,true];
 	_b setVariable ["ws_bUnits",_bu + 1,true];
-	_b setVariable ["ws_bPosLeft",_bpl,true];
+	_b setVariable ["ws_bPosLeft",_bpl,true]; 
 
 	if (_debug) then {_mkr = createMarker [format ["%1-bpos",_u],getPos _u];_mkr setMarkerSize [0.5,0.5];_mkr setMarkerType "mil_dot";_mkr setMarkerColor "ColorGreen";};
 };
 
+// If assignGear AI parameter is enabled, set the faction on the spawned units then pass them to assignGear AI.
+if (_assignGearFaction != "") then {
+	[units _grp,_assignGearFaction] call f_fnc_setVirtualFaction;
+	[units _grp] execVM "f\assignGear\f_assignGear_AI.sqf";
```

Comment:
strictly speaking this should probably be remote execed because assignGear_AI only works on the server. Though createGarrison almost always only runs on the server too, so meh.

Reply:
Documentation recommends running on headless client as well as server, so done.